python/BioSimSpace/Protocol/_free_energy.py

```diff
@@ -48,7 +48,8 @@ def __init__(self,
                  temperature=_Types.Temperature(300, "kelvin"),
                  pressure=_Types.Pressure(1, "atmosphere"),
                  report_interval=100,
-                 restart_interval=500
+                 restart_interval=500,
+                 pert_type="standard"
```

Comment:
Could this be named `perturbation_type` for clarity with `getPerturbationType` and `setPerturbationType` methods.

Reply:
see ab8be7b04c708df335aa173061156f0415e1be3b